Sources/hap-server/main.swift

```diff
@@ -11,6 +11,26 @@ fileprivate let logger = Logger(label: "bridge")
     import Glibc
 #endif
 
+func createLogHandler(label: String) -> LogHandler {
```

Comment:
I guess this changes the log level depending on the release? Can you move that to a separate PR?

Reply:
OK moved to #127